particle_list.yml

```diff
@@ -1,821 +1,780 @@
 ParticleList:
   gamma:
     PID: 22
-    Mass: { Value: 0.0 }
+    Mass: 0
```

Comment:
I think we have to think about this to be sure we do the right thing here. Having this dict structure with `Value`, `Error` etc could help make it easier to create Parameter objects from this. On the other hand, Error does not make much sense here in terms of setting up a fit (minuit uses this for initial step sizes but its not necessary).

Reply:
In the current implementation, you can do both so that the YAML file becomes readable. That's also one of the reasons for https://github.com/ComPWA/expertsystem/pull/42#discussion_r433749950